src/pipe.js

```diff
@@ -4,6 +4,7 @@
     var scripts = doc.getElementsByTagName('script');
     // State to maintain if all fragments on the page are initialized
     var initState = [];
+    var ranges = [];
```

Comment:
Also IMO we can use the existing initState instead of having one more state for checking the initialisation of fragments.